packages/tsconfig/tsconfig.app.json

```diff
@@ -2,10 +2,9 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "allowJs": true,
-    // Allow spreading Maps/Sets into arrays
-    "downlevelIteration": true,
```

Comment:
This isn't necessary, as we've long-since been running on modern ES6+ environments.